mix.exs

```diff
@@ -6,7 +6,7 @@ defmodule Holidefs.Mixfile do
   def project do
     [
       app: :holidefs,
-      version: "0.3.2",
+      version: "0.4.0",
```

Comment:
Why a minor version bump? Couldn't we just bump the patch version to `0.3.3`?

Reply:
Figured adding a locale is like adding functionality, not just a bug fix 🤷🏻‍♂️

Reply:
Yeah, but it's totally backward compatible feature, so IDK